components/configurator/view/ConnectConfiguratorView.js

```diff
@@ -68,7 +72,8 @@ const ConnectConfiguratorView = () => {
         console.log(e) // eslint-disable-line no-console
       }
     } else {
-      connect()
+      setLoading(true)
+      connect().finally(() => setLoading(false))
```

Comment:
I could add an arbitrary delay here to prevent the loading state "flash" from occurring if the `connect()` call is very fast. With the delay it would display for at least a second or more.

Reply:
https://user-images.githubusercontent.com/2985457/165366115-366e91c7-2f58-40d0-b6b3-e9320f4c554e.mov

Reply:
Actually adding an arbitrary delay here won't do much because we switch to the next view before this resolves completely. We may need to revisit that when we look into improving the NFT loading experience that you mentioned @mckelvey.

Reply:
That seems reasonable @seanconnollydev... let’s revisit.